### PR TITLE
Fixing testcase related to functions

### PR DESCRIPTION
**Pull request by abimarank**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: tests/ballerina-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java [hunk 1]
```diff
@@ -32,14 +32,14 @@
...
-        BAssertUtil.validateError(compile, 0, "redeclared symbol 'foo'", 5, 1);
+        BAssertUtil.validateError(compile, 0, "redeclared symbol 'foobar'", 5, 1);
     }
 
     @Test(description = "Test defining ballerina function with duplicate parameters")
     public void testDuplicateParameters() {
         CompileResult compile = BCompileUtil.compile("test-src/functions/duplicate-parameters.bal");
         Assert.assertEquals(compile.getErrorCount(), 1);
-        BAssertUtil.validateError(compile, 0, "redeclared symbol 'param'", 1, 28);
+        BAssertUtil.validateError(compile, 0, "redeclared symbol 'param'", 1, 31);
```
Comment (afkham) [inline tests/ballerina-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java]: Can you explain what this 28 & 31 are?

Comment (abimarank) [reply tests/ballerina-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java]: I changed the function name from `foo` to `foobar` in duplicate-function.bal file like following

```
function foobar(string param, int param) {
	string a;
	a = param;
}
```
You can seea duplicate args with `param` in 31 line.

Comment (afkham) [reply tests/ballerina-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java]: Can you put that as a comment in the test?

Comment (abimarank) [reply tests/ballerina-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java]: Done.

**Pull request merged**